saleor/graphql/app/tests/queries/test_app_extension.py

```diff
@@ -412,7 +412,7 @@ def test_app_extension_with_app_query_by_staff_with_permissions(
 
     # when
     response = staff_api_client.post_graphql(
-        QUERY_APP_EXTENSION_WITH_APP, variables, permissions=[permission_manage_apps]
```

Comment:
I don't think there is a need to leave that test, and especially with such a name.
I think just removing `without_permissions` from names of 2 tests and reducing the redundant ones should be okay.